lab-davide/lib/greet.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+module.exports = function(name){
```

Comment:
this function should only be return 'hello' + name,
because thats what the instuctions are asking for...
you dont want to add extra code to the instructions